PennyPincherExample/PennyPincherExample/ViewController.swift

```diff
@@ -52,8 +49,61 @@ class ViewController: UIViewController, UITextFieldDelegate {
         recognizerResultLabel.text = "Template: \(template.id), Similarity: \(similarityString)"
     }
    
+    //MARK:- Actions
+    @IBAction func didTapAddTemplate(_ sender: AnyObject) {
+        if
+            let text = templateTextField.text,
+            let template = PennyPincher.createTemplate(text, points: gestureView.points) {
```

Comment:
I think that makes sense. There's no need to update the view if you fail to alter the model.